spec/addressable/uri_spec.rb

```diff
@@ -5515,6 +5515,47 @@ def to_s
   end
 end
 
+describe Addressable::URI, "When set the basename " do
+  it "should have the correct basename: 'index.html'" do
+    uri = Addressable::URI.parse("http://test.com")
+    uri.basename = "index.html"
+
+    expect(uri.basename).to eq("index.html")
+    expect(uri.to_s).to eq("http://test.com/index.html")
+  end
+end
+
+describe Addressable::URI, "When change the basename " do
+  it "should have the new basename: 'item'" do
+    uri = Addressable::URI.parse("http://test.com/some/path/index.html")
```

Comment:
Same issue as above. This text context is really similar to others in the test suite. You should append to those `describe` blocks rather than starting a new one.